flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -529,6 +529,18 @@ public void killCluster(ApplicationId applicationId) throws FlinkException {
                         "Hadoop security with Kerberos is enabled but the login user "
                                 + "does not have Kerberos credentials or delegation tokens!");
             }
+
+            boolean fetchToken =
+                    flinkConfiguration.getBoolean(SecurityOptions.KERBEROS_FETCH_DELEGATION_TOKEN);
+            boolean yarnAccessFSEnabled =
```

Comment:
We could use `CollectionUtil.isNullOrEmpty` to check whether `YARN_ACCESS` is not set.

Reply:
Done